truss/config/trt_llm.py

```diff
@@ -3,9 +3,8 @@
 from enum import Enum
 from typing import Optional
 
-from pydantic import BaseModel, field_validator, model_validator
+from pydantic import BaseModel, field_validator
```

Comment:
I didn't find `field_validator` in V1 API either - can you double check please?

Reply:
Ah you're right. This is what we had before. Afaik this hasn't caused any issues, but I'm assuming there's some combination of python version/packages that will force v1 to be installed and then we'll run into problems?

Reply:
Talked with @marius-baseten and I will fix this in a follow up

Reply:
Customer issues on this might be delayed: it's an unfortunate corner case to use v1, most do already use v2, and it would only surface once one of those few v1-users updates truss.

That doesn't mean it's not a potential issue and would should be careful...